plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui.test/META-INF/MANIFEST.MF

```diff
@@ -10,6 +10,9 @@ Fragment-Host: com.google.cloud.tools.eclipse.appengine.deploy.ui
 Require-Bundle: org.hamcrest;bundle-version="1.1.0",
  org.junit;bundle-version="4.12.0"
 Import-Package: com.google.cloud.tools.eclipse.appengine.facets,
+ com.google.cloud.tools.eclipse.test.util,
+ com.google.cloud.tools.eclipse.test.util.ui,
+ com.google.common.base;version="15.0.0",
```

Comment:
I changed it to 17. We have guava 17 bundled in `com.google.cloud.tools.eclipse.3rdparty.feature`. If we want to update to 20, we can do it (unless there's some dependency on 17) but should be done in a separate PR.